cmake/cpp_sc2.cmake

```diff
@@ -0,0 +1,22 @@
+message(STATUS "FetchContent: cpp-sc2")
```

Comment:
Please use the same folders structure as in cpp-sc2, i.e. this one should be thirdparty/... as cpp-sc2 is external dependency for this bot.